doc/performance-investigations/0003-session-performance/summary.md

```diff
@@ -0,0 +1,70 @@
+# Use Redis for storing Questionnaire Runner sessions
+
+The EQ session data are currently stored in Datastore. It may be more performant to store these data in Redis (which is already used for JWT).
```

Comment:
- This should be either "EQ sessions are currently" or "EQ session data is currently".
- "store these data" -> "store this data"